useful_resources/sample_code/ngpvan_sample_list.py

```diff
@@ -0,0 +1,73 @@
+# ### METADATA
+
+# Connectors: VAN
+# Description: Creates a new saved list from a random sample of an existing saved list in VAN
+
+# ### CONFIGURATION
+
+# Set the configuration variables below or set environmental variables of the same name and leave
+# these with empty strings.  We recommend using environmental variables if possible.
+
+config_vars = {
+    # VAN
+    "VAN_API_KEY": "",
+    "VAN_DB_NAME": "",  # One of MyVoters, MyCampaign, MyMembers, EveryAction
+    "VAN_FOLDER_ID": "",  # Can be found in the URL on a Folder page in the FolderID param
+    "VAN_SAVED_LIST_ID": "",
+    "VAN_SAMPLE_LIST_SIZE": "",  # Must be a string
+    "VAN_SAMPLE_LIST_NAME": "",
+    # AWS
+    "AWS_ACCESS_KEY_ID": "",
+    "AWS_SECRET_ACCESS_KEY": "",
+    "S3_BUCKET": "",  # Bucket that the sample list data should be uploaded to
+}
+
+
+# ### CODE
+
+from parsons import VAN  # noqa: E402
+from parsons import logger  # noqa: E402
+import os  # noqa: E402
+import random  # noqa: E402
+
+# Setup
+
+# If variables specified above, sets them as environmental variables
+for name, value in config_vars.items():
+    if value.strip() != "":
+        os.environ[name] = value
+
+van = VAN(db=os.environ["VAN_DB_NAME"])
+
+# Get details on the saved list we're pulling from and the folder we're uploading to
+saved_list = van.get_saved_list(os.environ["VAN_SAVED_LIST_ID"])
+folder = van.get_folder(os.environ["VAN_FOLDER_ID"])
+
+saved_list_download = van.download_saved_list(os.environ["VAN_SAVED_LIST_ID"])
+
+# Generate a random sample of VAN IDs from the list
+saved_list_sample_ids = random.sample(
+    saved_list_download["VanID"],
```

Comment:
I'm not 100% positive if this would be true on the results of a saved list download, but I believe VAN defaults to camelCase.

Reply:
This worked when I ran it against VAN in our test instance so not sure if there's an issue with different versions?

Reply:
I don't think there should be issues with versions. If this casing worked, then we should probably ignore any of my comments on casing!

Reply:
Is the VAN connector, or Parsons in general, case sensitive when it comes to column names? I was trying to figure this out yesterday when working with `map_and_coalesce_columns` and ended up guessing it was case-insensitive. Whichever it is, we should say so prominently in the documentation. @eliotst?